<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GiveOrderDiag</class>
 <widget class="QDialog" name="GiveOrderDiag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Issue off diagnostics</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/icons/icons/report_magnify_go.png</normaloff>:/icons/icons/report_magnify_go.png</iconset>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="labelAlignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Order #:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="eOrderID">
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>71</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Date:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="eDate">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="displayFormat">
        <string>dd.MM.yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>24</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Issued by:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="eGiver">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Inspection 
 results:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPlainTextEdit" name="eInspect">
     <property name="documentTitle">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Identified 
 defects:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPlainTextEdit" name="eDefects"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Recommendations 
 for repair:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPlainTextEdit" name="eRecomm"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="bCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/icons/icons/door_out.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="cPreview">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Preview</string>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bPrint">
       <property name="text">
        <string>Print</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/icons/icons/printer.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbIssuedDiag">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sets corresponding order state&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Issued with diag.</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bSave">
       <property name="text">
        <string>Save and close</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/icons/icons/page_save.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>eOrderID</tabstop>
  <tabstop>eDate</tabstop>
  <tabstop>eGiver</tabstop>
  <tabstop>eInspect</tabstop>
  <tabstop>eDefects</tabstop>
  <tabstop>eRecomm</tabstop>
  <tabstop>bPrint</tabstop>
  <tabstop>bSave</tabstop>
  <tabstop>bCancel</tabstop>
  <tabstop>cPreview</tabstop>
 </tabstops>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
